source/features/index.js

```diff
@@ -0,0 +1,181 @@
+		fn: require('./likes-button-navbar').default,
+		init: true
+	},
+	cleanNavbarDropdown: {
+		id: 'feature-clean-navbar-dropdown',
+		category: 'header',
+		label: 'Cleanup navbar dropdown',
+		fn: require('./clean-navbar-dropdown').default
+	},
+	cleanupSearchSuggestions: {
+		id: 'feature-cleanup-search-suggestions',
+		category: 'header',
+		label: 'Remove hashtags suggestions in the search popover',
+		init: true
+	},
+	hideMomentsTab: {
+		id: 'feature-hide-moments-tab',
+		category: 'header',
+		label: 'Hide "Moments" tab',
+		init: true
+	},
+	hideTwitterLogo: {
+		id: 'feature-hide-twitter-logo',
+		category: 'header',
+		label: 'Hide Twitter bird logo',
+		init: true
+	},
+
+	/* HOME */
+	hideHomeFooterCard: {
+		id: 'feature-home-hide-footer-card',
+		category: 'home',
+		label: 'Hide "Footer" Card',
+		init: true
+	},
+	hideHomeProfileCard: {
+		id: 'feature-home-hide-profile-card',
+		category: 'home',
+		label: 'Hide "Profile" Card',
+		init: true
+	},
+	hideHomeTrendsCard: {
+		id: 'feature-home-hide-trends-card',
+		category: 'home',
+		label: 'Hide "Trends" Card',
+		init: true
+	},
+	hideHomeWhoToFollowCard: {
+		id: 'feature-home-hide-who-to-follow-card',
+		category: 'home',
+		label: 'Hide "Who To Follow" Card',
+		init: true
+	},
+
+	/* NOTIFICATIONS */
+	hideNotificationsFollowActivity: {
+		id: 'feature-notifications-hide-follow-activity',
+		category: 'notifications',
+		label: 'Hide "Followed you" activity',
+		init: true
+	},
+	hideNotificationsListActivity: {
+		id: 'feature-notifications-hide-list-activity',
+		category: 'notifications',
+		label: 'Hide "Added you to a list" activity',
+		init: true
+	},
+
+	/* PROFILE */
+	hideProfileHeader: {
+		id: 'feature-remove-profile-header',
+		category: 'profile',
+		label: 'Hide the header image on profile pages',
+		fn: require('./remove-profile-header').default
+	},
+	disableCustomColors: {
+		id: 'feature-disable-custom-colors',
+		category: 'profile',
+		label: 'Use your personal color theme on all profiles',
+		fn: require('./disable-custom-colors').default
+	},
+
+	/* TIMELINE */
+	autoLoadNewTweets: {
+		id: 'feature-auto-load-new-tweets',
+		category: 'timeline',
+		label: 'Auto-loads new tweets in the stream if you\'re scrolled to the top',
+		fn: require('./auto-load-new-tweets').default
+	},
+	codeHighlight: {
+		id: 'feature-code-highlight',
+		category: 'timeline',
+		label: 'Syntax highlighting in code blocks',
+		fn: require('./code-highlight').default
+	},
+	hideFollowTweets: {
+		id: 'feature-hide-follow-tweets',
+		category: 'timeline',
+		label: 'Hide "And others follow" tweets in the stream',
+		fn: require('./hide-follow-tweets').default
+	},
+	hideLikeTweets: {
+		id: 'feature-hide-like-tweets',
+		category: 'timeline',
+		label: 'Hide "Liked" tweets in the stream',
+		fn: require('./hide-like-tweets').default
+	},
+	hidePromotedTweets: {
+		id: 'feature-hide-promoted-tweets',
+		category: 'timeline',
+		label: 'Hide promoted tweets',
```

Comment:
No reason to have this be configurable.

Reply:
Right

Reply:
The point of it being configurable is to let people configure it the way they want. I promote tweets for my company and I want to be able to see how they look. If this is not configurable then I can't use this extension and all the effort in this PR is back to square 1.

Reply:
Yeah, the reason why I made all configurable is because I also believe this. People should be able to only enable what they want. The extension should not get in the way.

Reply:
Ok, leave this configurable then.